### ISSUE TITLE
code blocks via ``` are annoying in template functions

### ISSUE THREAD

**Issue by glenjamin** [glenjamin/devcards-js#12]
eg.
```js
devcard.anon(
  `
  Cards don't have to have names
  \`\`\`js
  devcard.anon(doc, body)
  \`\`\`
  `,
  <h3>What's in a name, anyway?</h3>
);
```

Would be good to provide some alternative that's a bit nicer.

**Comment by glenjamin** [comment]
Oh, apparently this isn't actually needed: https://gist.github.com/jorendorff/d3df45120ef8e4a342e5

**Comment by Yomguithereal** [comment]
We could work a bit on the Markdown parser to provide another way to do so like by using `~` for instance, but we would be taking a step away from Markdown syntax.

**Comment by glenjamin** [comment]
~~~js
"I just learned something new, github already allows ~~~"
~~~

**Comment by glenjamin** [comment]
Results look good: http://johnmacfarlane.net/babelmark2/?text=~~~js%0AWhat+up%3F%0A~~~

**Comment by glenjamin** [comment]
It's even in the commonmark spec http://spec.commonmark.org/0.24/#fenced-code-blocks

**Comment by Yomguithereal** [comment]
I guess this is one problem solved.

**Comment by glenjamin** [comment]
Yup, I'll close once the examples are all converted to use ~~~

**Comment by glenjamin** [comment]
Closed via cc666ec
